miniGithub/templates/miniGithub/base.html

```diff
@@ -11,8 +11,9 @@
       <div class="container">
         <ul class="nav navbar-nav">
 {# Django Home Link / Admin Link / Register Link#}
-          <li><a class="navbar-brand" href="{% url 'projects' %}">DJANGO</a></li>
+          <li><a class="navbar-brand" href="{% url 'projects' %}">PROJECT</a></li>
             {% if user.is_authenticated %}
+             <li><a class="navbar-link" href="{% url 'show_collaborator_projects'  %}">Collaboration projects</a></li>
```

Comment:
Maybe it made more sense to show this list in same page where list of own projects is displayed, instead of creating new page

Reply:
I think that is not a good idea because project that i am created are not the same like project that i'm working like collaborator, also i have not same privilege on them so we should't put them on same page